content/articles/why-data-storage-is-shifting-to-the-blockchain/index.md

```diff
@@ -0,0 +1,60 @@
+## Why Data Storage is shifting to the blockchain
+
+Some years ago, the most embraced form of data storage was the traditional in-house data centers. But it was not too long before cloud storage started to replace them. Cloud storage came with the promise of greater efficiency and more data storage capacity that attracted many businesses.
```

Comment:
The most embraced form of data storage was the traditional in-house data centers in the recent past. However, cloud storage started to replace it. Cloud storage came with the promise of greater efficiency and more data storage capacity that attracted many businesses.
```suggestion
The most embraced form of data storage was the traditional in-house data centers in the recent past. However, cloud storage started to replace it. Cloud storage came with the promise of greater efficiency and more data storage capacity that attracted many businesses.
```